packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes/ThemeContext';
+
+export enum SliderOrientation {
+  horizontal = 'horizontal',
+  vertical = 'vertical',
+}
+
+export interface Props {
+  min: number;
+  max: number;
+  orientation?: SliderOrientation;
+  // Set current positions of handle(s). If only 1 value supplied, only 1 handle displayed.
+  value?: number[];
+  reverse?: boolean;
+  formatTooltipResult?: (value: number) => number | string;
+  onChange?: (values: number[]) => void;
+}
+
+const defaultProps = { orientation: SliderOrientation.horizontal };
```

Comment:
You don't need to define default props for components like this. Instead, you can assign default values when destructing the properties, like this:

```
export const Slider: FunctionComponent<Props> = ({
  min,
  max,
  onChange,
  orientation = 'horizontal',
  reverse,
  formatTooltipResult,
  value,
})
```